### DISCUSSION TITLE
Percentage of battery as graph ?

### DISCUSSION THREAD

**Discussion by pietnie** [solectrus/solectrus#1826] [Ideas]
First of all: Great work and thanks a lot for your work. It runs without any issues on my Synology.

I would like to have a graph to see the percentage of the current battery status, so when I click on the current battery percentage that I can see how the battery was charged and (more important) how long in the night the battery was still in use. I found this the most helpful display in the old senec app before it was deactivated.

**Comment by ledermann** [comment]
Thanks, I'm glad you like it!

Battery diagram - there is already an issue about this, please look here: https://github.com/solectrus/solectrus/issues/1734

In short, this diagram will be implemented in one of the next updates.
